src/psyclone/transformations.py

```diff
@@ -2845,6 +2850,20 @@ def validate(self, nodes, options=None):
 
         super(ProfileRegionTrans, self).validate(nodes, options)
 
+        if options:
+            try:
+                name = options["profile_name"]
+                if not isinstance(name, tuple) or not len(name) == 2 or \
```

Comment:
pylint complains about this `if` too.

Reply:
I've suppressed the warnings as I don't see how the code would be better if it were split.